UPGRADING.rst

```diff
@@ -20,6 +20,13 @@ reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
 
+#7468  Update Swagger to v5.29.4
+================================
+
+Read the instructions in section 3.2.2 of the README and make the necessary
+updates.
```

Comment:
This is insufficient. While it's OK to refer to the README section, the README explains the *initial* setup. Describe what needs to be *changed*.